rest/route/user_test.go

```diff
@@ -870,12 +872,25 @@ func TestRenameUser(t *testing.T) {
 			}
 			assert.NoError(t, db.InsertMany(patch.Collection, p1, p2))
 
+			someOtherUser := user.DBUser{
+				Id:           "some_other_me",
+				EmailAddress: "me@awesome.com",
+				APIKey:       "my_key",
+				PatchNumber:  7,
+				Settings: user.UserSettings{GithubUser: user.GithubUser{
+					UID: 0, // Verify there's no issue inserting an empty UID if there's another empty UID
```

Comment:
Maybe I'm confused, but wouldn't this always pass since there's no unique index? I don't think we build unique indexes on the users collection for tests, no? So we aren't actually verifying if uniqueness is compatible with an empty UID, because this will pass the assertion regardless of whether the unique index is there or not.